### ISSUE TITLE
Define MTA configurations in project workspace (feat)

### ISSUE THREAD

**Issue by rmarting** [windup/intellij-mta#222]
Each MTA Configuration is created in the $HOME/.mta/tooling/model.json file, a shared folder for any IntelliJ instance.

If you have different projects where you want to use the MTA extension, at the end you have a single view of all the MTA configurations, not related with the current project opened in IntelliJ. This is behaviour could impact in the user experience as in the same view there are many different configurations, related or no with the current project opened.

This screenshot shows a list of MTA configurations not related with the current project opened. 

![mta-shared-configurations](https://user-images.githubusercontent.com/8723668/161296012-c563916e-b500-4444-b77c-1c50c90c496a.png)

IMHO it could be very usefull to save the MTA configurations at the level of the project folder ($PROJECT_HOME/.idea), or at least allow user to choose where to save the configuration. This feature will the developer could decide where to save the MTA analysis and organize them under he/she convenience.

Does it make sense?

**Comment by johnsteele** [comment]
@rmarting Agreed. The state location could be improved. Let's start with your suggestion of using $PROJECT_HOME/.idea
